### PR TITLE
Login page

### PR DESCRIPTION
**Pull request by pearlhamilton**
Screen Shot 2022-02-05 at 21 11 03

@dri94 for some reason the about page said I changed it to say OpenEdx - weird, so changed it back to say EmpowrX. Image needs adding too, have used a placeholder because I was having issues doing it

### CODE REVIEW

File: lms/templates/index_overlay.html [hunk 1]
```diff
@@ -35,7 +35,7 @@ <h1>How we will support you</h1>
         <div class="row gx-2">
             <div class="col">
                 <div class="p-3">
-                    <img class="img-responsive" src="https://d2l03dhf2zcc6i.cloudfront.net/images/networking.jpeg"
+                    <img class="img" src="https://d2l03dhf2zcc6i.cloudfront.net/images/networking.jpeg"
```
Comment (dri94) [inline lms/templates/index_overlay.html:38]: any reason this was changed?

**Review by dri94 — APPROVED** [review]
Amazing!

File: lms/templates/index_overlay.html (hunk 1 above)
Comment (pearlhamilton) [reply lms/templates/index_overlay.html:38]: the class was making the two images not stack when on a smaller screen. The rest of the images don't have this class

**Comment by pearlhamilton** [comment]
you will have to merge @dri94  :) I;m not authorised

**Pull request merged**